packages/forms/src/__tests__/FormikCheckboxField.spec.tsx

```diff
@@ -41,6 +42,42 @@ test('handles changes', async () => {
   expect(handleSubmit).toHaveBeenLastCalledWith({ agree: false });
 });
 
+test('format and parse value', async () => {
+  const handleSubmit = jest.fn();
+
+  const wrapper = renderFormField(
+    <FormikCheckboxField
+      name="status"
+      label="Status"
+      format={(value) => value === 'active'}
+      parse={(_, checked) => (checked ? 'active' : 'inactive')}
+    />,
+    {
+      initialValues: { status: 'active' },
+      onSubmit: handleSubmit,
+    },
+  );
+
+  const field = wrapper.getByLabelText('Status');
+
+  expect(field).toBeChecked();
+
+  MockEvent.click(field);
+  act(() => {
+    fireEvent.blur(field);
+  });
+
+  expect(field).not.toBeChecked();
+
+  wrapper.submitForm();
+
+  await waitFor(() => {
+    expect(handleSubmit).toHaveBeenCalledTimes(1);
+  });
+
+  expect(handleSubmit).toHaveBeenLastCalledWith({ status: 'inactive' });
```

Comment:
can you check also for the `false` value result?

Reply:
You mean with for `parse/format`?

Reply:
I meant `true`, just click to the checkbox one more time and check submission result